examples/dynamics/shot_at_an_angle_to_the_horizon.py

```diff
@@ -15,6 +16,10 @@
 
 angle_of_shot = Symbol("angle_of_shot")
 
+# At the highest point of the trajectory, the bullet velocity modulus is equal to
```

Comment:
It's wrong. At the highest point there is no vertical component of the velocity, therefore we can use only horizontal component, which is projection of initial velocity on the horizon.